data/tool_pages/platform/2in_00_the_ocaml_infrastructure.md

```diff
@@ -0,0 +1,108 @@
+
+## What is OCurrent?
+
+[OCurrent](https://ocurrent.org) is an OCaml eDSL for creating processing pipelines that automatically adjust when their inputs change. For example, when a Git repository is updated, OCurrent pipelines can automatically rebuild, test, and deploy the changes without manual intervention.
+
+## OCaml Infrastructure Services
+
+The OCaml infrastructure services can be grouped into three main categories: Community infrastructure services, services for individual projects, and infrastructure operation services. Here we present a brief overview of the major services from the first two categories.
+
+### Services for Individual Projects
+
+These services used for your own projects to help with reproducible builds, testing, benchmarking, and continuous integration:
+
+#### Docker Base Images
+
+**Service:** [docker-base-images](https://github.com/ocurrent/docker-base-images)
+
+**Website:** [images.ci.ocaml.org](https://images.ci.ocaml.org)
+
+This service builds the official `ocaml/opam` Docker images for various Linux distributions, OCaml versions, compiler flags, and architectures (including x86, ARM, PowerPC, s390x, and RISC-V). These images provide a consistent environment for development and testing and are used by many other CI services.
+
+**Using the pre-built OCaml Docker images:** You can view the available [pre-built Docker images for various OCaml configurations at DockerHub](https://hub.docker.com/r/ocaml/opam).
+
+#### OCaml CI
+
+**Service:** [ocaml-ci](https://github.com/ocurrent/ocaml-ci)
+
+**Website:** [ocaml.ci.dev](https://ocaml.ci.dev)
+
+A CI service for OCaml projects hosted on GitHub. It automatically tests projects against multiple OCaml versions and OS platforms by examining your opam files to determine build requirements.
+
+**How to enable OCaml CI on your GitHub project:** Check out the ["Getting Started" guide on the OCaml CI website](https://ocaml.ci.dev/getting-started).
+
+#### Continuous Benchmarking
+
+**Service:** [current-bench](https://github.com/ocurrent/current-bench)
+
+**Website:** [bench.ci.dev](https://bench.ci.dev)
+
+Provides continuous benchmarking for OCaml projects to track performance across different commits and branches.
+
+**How to enable current-bench on your GitHub project:** Check out the [current-bench User Manual](https://github.com/ocurrent/current-bench/blob/main/doc/user_manual.md).
+
+#### OCaml-Multicore-CI
+
+**Service:** [ocaml-multicore-ci](https://github.com/ocurrent/ocaml-multicore-ci)
+
+**Website:** [ocaml-multicore.ci.dev](https://ocaml-multicore.ci.dev)
+
+Tests OCaml projects against multicore OCaml versions - particularly useful if you want to ensure your code is compatible with OCaml 5.x.
+
+**How to enable OCaml-Multicore-CI on your GitHub project:** Check out the [OCaml-Multicore-CI README on GitHub](https://github.com/ocurrent/ocaml-multicore-ci).
```

Comment:
I have not been involved with this, but I don't know what its use case is beyond ocaml-ci now that 5 is out for some time and we are running ocaml 5.x in ocaml-ci.

It doesn't seem to me that this is a service we want people thinking is maintained and available for their use on individual projects.